modules/serverless-eventing-components.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * serverless-knative-eventing.adoc
+
+[id="serverless-eventing-components_{context}"]
+= Knative Eventing components
+
+A typical Knative Serving deployment on {ServerlessProductName} follows this basic workflow.
+
+. Event producers integrated with {ServerlessProductName} send information about changes in the state of the system to an event source.
+. The event source translates this information into an event, which is then passed to the broker. Events can also be passed directly from an event channel to a broker.
+. Once an event has been sent to the broker, triggers can filter which events are routed to a Subscription.
+
+Using the broker and trigger Kubernetes Custom Resource Definitions (CRDs) together enables an event delivery mechanism that hides the details of event routing from the event producer and event consumer.
+
+Event source:: Event sources are Kubernetes custom resources which provide a mechanism for registering interest in a class of events from an event producer.
++
+{ServerlessProductName} can be used with Kafka and Camel K event sources, which can be installed using the Operator Hub in the {product-title} web console.
+
+Broker:: A broker represents an event mesh. Events are sent to the broker's ingress and are then sent to any event consumers that are subscribed to that broker.
+Once inside the broker, metadata for the event is stripped away, so that there is no information about how the event entered the broker.
++
+Example YAML file defining a broker:
+----
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Broker
+metadata:
+  name: default <1>
+spec:
+  channelTemplateSpec: <2>
+    apiVersion: messaging.knative.dev/v1alpha1
+    kind: InMemoryChannel
+----
+
+<1> The `default` broker is provided with your {ServerlessProductName} Knative Eventing installation.
+<2> Event channels can be specified in the `channelTemplateSpec`.
+
+// NOTE: this is being replaced. Update when details are available upstream.
+
+
+Trigger:: A trigger enables a Knative service to subscribe to events from a specific broker.
++
+Example YAML file defining a trigger:
+----
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Trigger
+metadata:
+  name: my-service-trigger <1>
+spec:
+  broker: default <2>
```

Comment:
we lack filters here

Reply:
I'll add that with the Trigger module and filtering explanation/examples